### PR TITLE
PID: Improve the API docs and change default value of antiwindup

### PR DESCRIPTION
**Pull request by christophfroehlich**
- improve function docs
- add [[nodiscard]] to computeCommand

### CODE REVIEW

File: include/control_toolbox/pid.hpp [hunk 1]
```diff
@@ -113,12 +113,34 @@ class CONTROL_TOOLBOX_PUBLIC Pid
...
+   * \param d The derivative gain.
+   * \param i_max The max integral windup.
+   * \param i_min The min integral windup.
+   *
+   * \throws An std::invalid_argument exception is thrown if i_min > i_max
+   */
     Gains(double p, double i, double d, double i_max, double i_min)
     : p_gain_(p), i_gain_(i), d_gain_(d), i_max_(i_max), i_min_(i_min), antiwindup_(false)
     {
     }
```
[the remark this answers is not in the document]
Comment (saikishor) [reply include/control_toolbox/pid.hpp:130]: I think it is good to change it to true as well by default

**Review by saikishor — APPROVED** [review]
The rest looks spot on!

File: include/control_toolbox/pid.hpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (christophfroehlich) [reply include/control_toolbox/pid.hpp]: done

**Review by saikishor — APPROVED** [review]
Awesome. Thanks for the change!

**Pull request merged**